tpgtools/api/cloudbuildv2/samples/ghe_connection.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 Google LLC. All Rights Reserved.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: ghe_connection
+description: Creates and updates a GitHub Enterprise connection.
```

Comment:
Could we add an update step, or is there something blocking doing so? Generally, we try to exercise as many updatable fields as possible. When that isn't, we cover every API update method possible and try to cover the interesting looking corner cases that we can.

The app* fields look like easy candidates, unless those values are special somehow. I'll defer to your judgement on other fields (but more coverage would be better!)

Reply:
Done. The tests from DCL (which now I'm copying exactly) include update steps.